concent_api/core/subtask_helpers.py

```diff
@@ -29,7 +30,7 @@
 logger = getLogger(__name__)
 
 
-def update_timed_out_subtask(subtask: Subtask) -> None:
+def _update_timed_out_subtask(subtask: Subtask) -> None:
```

Comment:
This should not be in this commit because you are changing name of a function here.

Reply:
Why this name was changed in general?

Reply:
I know but it is related with previous PR. It is not directly part of this task, but small improvement of previous. This change is not necessary, but after merging previous PR i decided it will be better as it is now. So i did it here, this is to small to separate PR and code looks better :)

Reply:
It is changed because this function is used only in `subtask_helpers.py`, called by two other functions in this file, so i wanted to mark it as 'private'